kernels/volk/volk_16i_branch_4_state_8.h

```diff
@@ -98,73 +94,67 @@ volk_16i_branch_4_state_8_a_ssse3(short* target, short* src0, char** permuters,
   xmm8 = _mm_load_si128((__m128i*)permuters[2]);
   xmm10 = _mm_load_si128((__m128i*)permuters[3]);
 
-  for(; i < bound; ++i) {
-    xmm5 = _mm_load_si128(p_src0);
-    xmm0 = _mm_shuffle_epi8(xmm5, xmm0);
-    xmm6 = _mm_shuffle_epi8(xmm5, xmm6);
-    xmm8 = _mm_shuffle_epi8(xmm5, xmm8);
-    xmm10 = _mm_shuffle_epi8(xmm5, xmm10);
+  xmm5 = _mm_load_si128(p_src0);
+  xmm0 = _mm_shuffle_epi8(xmm5, xmm0);
+  xmm6 = _mm_shuffle_epi8(xmm5, xmm6);
+  xmm8 = _mm_shuffle_epi8(xmm5, xmm8);
+  xmm10 = _mm_shuffle_epi8(xmm5, xmm10);
 
-    p_src0 += 4;
```

Comment:
Why did you remove this line? Shouldn't the source vector be moved in every iteration?

Also `ctest -V -R branch_4` revealed:
```
2: Did not run a test for kernel: volk_16i_branch_4_state_8 !
1/1 Test #2: qa_volk_16i_branch_4_state_8 .....   Passed    0.00 sec
```
In other words we do not have a test for this kernel which is a problem. Also, this makes it more difficult to verify if the changes here are correct.

Reply:
`p_src0` is no longer used after the line above `xmm5 = _mm_load_si128(p_src0);` ... so it's wasteful to manipulate it here.

Reply:
Grrr... about not having a test for this!

Reply:
I see, the loop was removed entirely. Yeah, in that case, this line can go.

Reply:
I will still rebase it.